internal/security/secretstore/init.go

```diff
@@ -164,34 +233,17 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 		lc.Info("no token provider configured")
 	}
 
-	// credential creation
-	gk := NewGokeyGenerator(absPath)
-	lc.Warn("WARNING: The gokey generator is a reference implementation for credential generation and the underlying libraries not been reviewed for cryptographic security. The user is encouraged to perform their own security investigation before deployment.")
-	cred := NewCred(req, absPath, gk, configuration.SecretService.GetSecretSvcBaseURL(), lc)
-
-	// Detour: Create secrets engine if not there already
-	vaultToken, err := GetAccessToken(absPath)
-	if err != nil {
-		lc.Error(fmt.Sprintf("failed to reload root token: %s", err.Error()))
-		os.Exit(1)
-	}
-	// Note: trailing slash on secret/ is required.
-	installed, err := vc.CheckSecretEngineInstalled(vaultToken, "secret/", "kv")
+	// Enable KV secret engine
```

Comment:
The following diff looks a little odd... what has really happened is that the code to enable the secrets engine has been out of lined to a private method.